tests/tas-mtls/src/test/java/org/alfresco/rest/MtlsRestTest.java

```diff
@@ -26,34 +34,29 @@
 @ContextConfiguration ("classpath:alfresco-mtls-context.xml")
 public abstract class MtlsRestTest extends AbstractTestNGSpringContextTests
 {
-    private static Logger LOGGER = LogFactory.getLogger();
+    protected static Logger LOGGER = LogFactory.getLogger();
 
     @Autowired
-    protected ServerHealth serverHealth;
-
-    @Autowired
-    protected RestWrapper restClient;
-
+    protected TestMtlsProperties testMtlsProperties;
     @Autowired
     protected DataUserAIS dataUser;
-
     @Autowired
-    protected DataSite dataSite;
-
-    @Autowired
-    protected DataContent dataContent;
```

Comment:
I've found it near impossible to reliably set up SSL for dataContent especially, we don't use dataSite at all and dataUser is just retrieved to get the admin user login and password for tests basic authorization.